content/go/concepts/strings/terms/compare/compare.md

```diff
@@ -0,0 +1,85 @@
+---
+Title: 'Compare()'
+Description: 'Compares two strings in lexicographical order.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Functions'
+  - 'Characters'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`strings.Compare()`** function is a built-in function in Golang that compares two strings lexicographically. It returns an integer value that indicates the relationship between the two strings.
+- If the two strings are equal, the function returns `0`.

```

Comment:
Like to move bulleted list to end of syntax section.

Reply:
@SSwiniarski I've made the changes, please check. Thank you.